view_presentation/interface/config.py

```diff
@@ -0,0 +1,3 @@
+
+
+gensim_path = '../../GoogleNews-vectors-negative300.bin'
```

Comment:
are there any other hardcoded parameters that could be made a config and thus live here? e.g.,:

- the set of interfaces of all available one wants to actually run
- interface-specific params?